src/file/error.cr

```diff
@@ -0,0 +1,41 @@
+class File < IO::FileDescriptor
+end
+
+class File::Error < IO::Error
+  getter file : String
```

Comment:
Would be nice to have some documentation on these.

Reply:
Of course. Documentation is still missing in many classes and methods.